.github/workflows/ci.yml

```
@@ -37,6 +37,10 @@ jobs:
             ~/.cargo/bin
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
+<<<<<<< HEAD
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)